DEPENDENCIES

```diff
@@ -98,11 +98,36 @@ Prerequisites for building the OpenDataPlane (ODP) API
      --with-openssl-path=/home/${USER}/src/install-openssl-aarch64
    $ make
 
```

Comment:
```
Signed-off-by: Govindarajan <govindarajan.mohandoss@arm.com>
               Malvika Gupta <Malvika.Gupta@arm.com>
               Matias Elo <matias.elo@nokia.com>
```

There needs to be "Signed-off-by:" (or reviewed-by, etc) before each name in commit log message.